app/models/product.rb

```diff
@@ -86,7 +86,7 @@ def available_for
   end
 
   def self.available_or_recommended_modules(root_product_ids)
-    joins(:product_extensions_associations).where(free: true, products_extensions: { root_product_id: root_product_ids }).or(
+    joins(:product_extensions_associations).where(free: true, product_type: :module, products_extensions: { root_product_id: root_product_ids }).or(
```

Comment:
same comment as in https://github.com/SUSE/happy-customer/pull/3705/files#r180096774